google-cloud-storage/samples/storage_configure_retries.rb

```diff
@@ -0,0 +1,50 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START storage_configure_retries]
+def configure_retries bucket_name: nil, file_name: nil
+  # The ID of your GCS bucket
+  # bucket_name = "your-unique-bucket-name"
+
+  # The ID of your GCS object
+  # file_name = "your-file-name"
+
+  require "google/cloud/storage"
+
+  # Customize retry with the maximum retry attempt of 5 (default=3).
```

Comment:
Can these comments be interspersed with the storage client being defined? So users know which comment belongs to which line. Node example of doing this here: https://github.com/googleapis/nodejs-storage/blob/HEAD/samples/configureRetries.js

Reply:
Mapped the comments to the code line they belong to.